README.md

```diff
@@ -93,30 +93,30 @@ Example:
 
 ```dart
 // Using String indices.
-String firstTagString(String source) {
-  var start = string.indexOf("<") + 1;
+String? firstTagString(String source) {
```

Comment:
I do wonder how we can best provide (lightweight, simple) analysis for snippets in readme files.